### PR TITLE
Add the loading="lazy" attribute to section about lazy loading

### PR DESCRIPTION
**Pull request by arohan0929**
#### Summary

Added another way to lazy load images using the `loading` attribute on the image tag. Rewrote the summary before the explanation to include the added way and explain what lazy loading is.

#### Motivation

Since the `loading` attribute is relatively new it is not documented everywhere. It is also very easy compared to lazy loading images using JavaScript which will help the readers save time.

#### Supporting details

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/loading/index.md [hunk 1]
```diff
@@ -108,7 +108,17 @@ This will remove the blur effect within half a second, which looks good enough f
 
 The image loading mechanism discussed in the above section works OK — it loads the images after rendering the HTML structure, and applies a nice transition effect in the process. The problem is that it still loads _all_ the images at once, even though the user will only see the first two or three upon page load.
 
-This problem can be solved with the new [Intersection Observer API](/en-US/docs/Web/API/Intersection_Observer_API) — using this we can ensure that images will be loaded only when they appear in the viewport.
+This problem can be solved by lazy loading the images. Lazy loading is a techinque to load images only when they appear in the viewport. We explained two ways to lazy load images below.
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/loading/index.md:111]:
```suggestion
This problem can be solved by loading the images only when needed: this is called _lazy loading_. [Lazy loading](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) is a technique to load images only when they appear in the viewport. There are several ways to tell the browser to lazy load images.
```

File: files/en-us/web/progressive_web_apps/loading/index.md [hunk 2]
```diff
@@ -108,7 +108,17 @@ This will remove the blur effect within half a second, which looks good enough f
...
+
+### The HTML {{htmlattrdef("loading")}} attribute
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/loading/index.md:113]:
```suggestion
### The loading attribute on \<img>
```

File: files/en-us/web/progressive_web_apps/loading/index.md [hunk 3]
```diff
@@ -108,7 +108,17 @@ This will remove the blur effect within half a second, which looks good enough f
...
+
+This is a way to lazy load images without JavaScript.
+
+Just add the attribute with the `lazy` value as shown:
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/loading/index.md:117]: Note that the attribute won't work if JavaScript is not activated. This makes the sentence ambiguous.
```suggestion
The easiest way to tell the browser to load lazily doesn't involve JavaScript. You add the [`loading`](/en-US/docs/Web/HTML/Element/img#attr-loading) attribute to an {{HTMLElement("img")}} element with the value `lazy`, and the browser will know to load this image only when needed.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by arohan0929** [comment]
Thank you!

**Pull request merged**